test/unit/utils/users.test.js

```diff
@@ -65,4 +66,31 @@ describe("users", function () {
       expect(participantUserId).to.include(userId);
     });
   });
+
+  describe("getUsernamesFromPRs", function () {
+    it("returns an array of unique usernames from the filtered PRs/Issues response", function () {
+      // Mock the allPRs data
+      const allPRs = filteredPRs;
+
+      // Execute the function and store the result
+      const usernames = usersUtils.getUsernamesFromPRs(allPRs);
```

Comment:
getUsernamesFromPrs is a function that return all the usernames in an array?? Correct me if I am wrong??

Reply:
yes